pages/blog/[uid].tsx

```diff
@@ -78,8 +71,8 @@ export async function getStaticProps({
       );
 
     const linkedAuthor =
```

Comment:
this should happen in the query above, there is no need to do a separate query here